src/mixins/swagger.mixin.ts

```diff
@@ -8,10 +8,14 @@ export interface SwaggerOptions {
     enumName?: string;
     format?: string;
     nullable?: boolean;
-    minimum?: number,
-    maximum?: number,
+    maxLength?: number,
+    maxItems?: number,
+    maxValue?: number,
+    minLength?: number,
+    minItems?: number,
+    minValue?: number,
     optional?: boolean,
-    type?: string,
+    type?: string | Constructor | Constructor[],
```

Comment:
I'm starting to realize the the mixin options should probably not be combined with the recipe options - we currently defined the recipes as accepting `MixinOptions & RecipeOptions`. I will probably revisit this soon that, for example, you can't pass `maxLength` to the array decorator.